Resources/views/snippets/common/userInput/form/formLoginStandard.html.twig

```diff
@@ -22,7 +22,7 @@
     {% endif %}
     {% if not bIsLoggedIn %}
         <div class="loginForm">
-            <form accept-charset="utf-8" method="post" class="{% block formClass%}form-horizontal{% endblock%}">
+            <form accept-charset="utf-8" method="post" action="{{ loginFormAction is defined ? loginFormAction : '?' }}" class="{% block formClass%}form-horizontal{% endblock%}">
```

Comment:
I'd omit the whole `action` attribute if loginFormAction is not defined. Adding a question mark will lead to subtle routing problems (and isn't correct per se, as we like to call the unmodified URL on submit).

Reply:
I was wondering if that (calling original url) was really intended.

As far as I know not specifying action ist considered an error in html.

Reply:
The action attribute was required before HTML 5 only. Omitting would also make clear that it's really intended to call the current URL again.

Reply:
ok